.github/workflows/taiko-client--hive_test.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   hive_tests:
-    if: github.event.pull_request.draft == false && contains(github.event.pull_request.labels.*.name, 'option.hive-test')
```

Comment:
This `option.hive-test` label check will skip `hive-test` in `taiko-client` related PRs. Do we want to turn on it? @dantaik

Reply:
At this PR just temporarily turn off this check for testing `hiv-test`.